src/app/@theme/components/footer/footer.component.ts

```diff
@@ -14,4 +16,7 @@ import { Component } from '@angular/core';
   `,
 })
 export class FooterComponent {
```

Comment:
Fix typo error from `xport` to `export`

Reply:
Hey there! Not quite sure what you mean by that? It all seems fine to me already

Reply:
![image](https://user-images.githubusercontent.com/16066485/43442724-77323014-94a7-11e8-826f-bb661d4a9af6.png)

Reply:
Hm...

Reply:
It is okay.

Reply:
Hi.
It is okay. Problem at displaying changes.